test/libraries/ProtocolFeeLibrary.t.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import "src/libraries/ProtocolFeeLibrary.sol";
+import "src/libraries/LPFeeLibrary.sol";
+import "forge-std/Test.sol";
+
+contract ProtocolFeeLibraryTest is Test {
+    function test_zeroForOne() public {
+        uint24 fee = uint24(ProtocolFeeLibrary.MAX_PROTOCOL_FEE) << 12 | ProtocolFeeLibrary.MAX_PROTOCOL_FEE;
+        assertEq(ProtocolFeeLibrary.getZeroForOneFee(fee), uint24(ProtocolFeeLibrary.MAX_PROTOCOL_FEE));
```

Comment:
for this test and the one below, could you make the 2 values different? because from this test we cannot check whether its getting the correct one or not
like
```
uint24(ProtocolFeeLibrary.MAX_PROTOCOL_FEE - 1) << 12 | ProtocolFeeLibrary.MAX_PROTOCOL_FEE;
```

then you know if its getting left or right number